readme.md

```diff
@@ -5,60 +5,61 @@ English ·
 [Français](./Translation-French.md) ·
 [Русский](./Translation-Russian.md) ·
 [Deutsch](./Translation-German.md) ·
-[日本語](./Translation-Japanese.md)
+[日本語](./Translation-Japanese.md) .
```

Comment:
You use wrong character as separator, it is *not* a regular dot.

Reply:
Fixed